public/pages/DetectorDetail/utils/helpers.tsx

```diff
@@ -107,3 +108,13 @@ export const getDetectorStateDetails = (
     </Fragment>
   );
 };
+
+export const containsIndex = (index: string, indices: CatIndex[]) => {
```

Comment:
Is it possible that `indices` is null?

Reply:
Good callout - I checked redux stores and confirmed the init values are empty arr (`[]`), and can only be overwritten by a valid set of indices from server-side. But just in case something is missed I think it's good to add a null check here - I'll add one